build/helper/documentation_helper.py

```diff
@@ -306,6 +308,17 @@ def _format_type_for_rst_documentation(param, config):
     return p_type
 
 
+rep_cap_method_desc = '''
+This method uses repeated capabilities (usually channels). If you call this method on the base session, then
```

Comment:
I think "base session" is confusing. Here's suggested edit:

```
This method uses repeated capabilities (usually channels). If called directly on the nidmm.Session object, then it will apply to the full session. You can specify what repeated capabilities to use using the Python
index notation:

.. code:: python

    session['0,1'].{0}({1})

```

I asked for feedback from @thejcannon and @MrElusive who are very familiar with NI-DCPower and Python.